packages/common/src/core/schemas/types.ts

```diff
@@ -151,6 +151,7 @@ export enum UiSchemaMessageTypes {
 
 export interface IConfigurationSchema extends JSONSchema {
   type?: "object";
+  selectionMode?: "string";
```

Comment:
hmm, just curious why we're adding this to the `IConfigurationSchema` interface? I don't _think_ we want to do this?

Reply:
Oops I left it here while I was testing the selection mode for the picker, no need to have it here anymore, removing...